Package fails to build

I am currently trying to build this package for melodic. I am using the latest release version: 1.7.6
When building the package I get the following error, which is causes the package build to fail.
```
CMake Error at /usr/share/cmake-3.14/Modules/FindBoost.cmake:2165 (message):
  Unable to find the requested Boost libraries.

  Boost version: 1.69.0

  Boost include path: /usr/include

  Could not find the following Boost libraries:

          boost_signals

  Some (but not all) of the required Boost libraries were found.  You may
  need to install these additional Boost libraries.  Alternatively, set
  BOOST_LIBRARYDIR to the directory containing Boost libraries or BOOST_ROOT
  to the location of Boost.
Call Stack (most recent call first):
  CMakeLists.txt:14 (find_package)


-- laser_assembler: 0 messages, 2 services
-- Configuring incomplete, errors occurred!
```
It seems like boost_signals is required, but boost does not have signals.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,13 +5,13 @@ project(laser_assembler)
 # Find dependencies
 ##############################################################################
 
-set(THIS_PACKAGE_ROS_DEPS 
+set(THIS_PACKAGE_ROS_DEPS
 tf sensor_msgs message_filters roscpp laser_geometry filters)
 
-find_package(catkin REQUIRED COMPONENTS 
-  ${THIS_PACKAGE_ROS_DEPS} 
+find_package(catkin REQUIRED COMPONENTS
+  ${THIS_PACKAGE_ROS_DEPS}
   message_generation)
-find_package(Boost REQUIRED COMPONENTS system signals)
+find_package(Boost REQUIRED COMPONENTS system)
 include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
 
 ##############################################################################
@@ -85,7 +85,7 @@ endif()
 # Install
 ##############################################################################
 
-install(TARGETS 
+install(TARGETS
     laser_scan_assembler_srv
     laser_scan_assembler
     merge_clouds

